containers/python/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM clipper/py-rpc:latest
 MAINTAINER Dan Crankshaw <dscrankshaw@gmail.com>
```

Comment:
This won't work because we can't reference files outside of the docker file's build context. The context directory (the one containing the docker file) must reference `clipper_admin` as a subdirectory.